### PR TITLE
Add week 8 content

### PR DESCRIPTION
**Pull request by KaranKamath**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -0,0 +1 @@
+.DS_Store
```
Comment (damithc) [inline .gitignore]: Remove

**Comment by damithc** [comment]
Minor change requested.

**Comment by KaranKamath** [comment]
Removed and rebased.
Is this because .gitignore should be updated in its own fix, or a standards violation?

**Comment by damithc** [comment]
My comment was not about `.gitignore` file. I was referring to a hidden Mac file (`.DS_Store` or something like that)

**Comment by damithc** [comment]
Actually, my mistake. I confused file name with file content.

**Pull request merged**